scripts/minify-css.js

```diff
@@ -39,8 +39,8 @@ postcss([
         // We need to extract the Tailwind screen size selectors (e.g. sm, md, lg)
         // so that we do not strip them out. As long as a class name appears in the HTML
         // in its entirety, PurgeCSS will not remove it.
-        // Ex. https://tailwindcss.com/docs/controlling-file-size/#writing-purgeable-html
-        defaultExtractor: content => content.match(/[\w-/:]*[\w-/:]/g) || [],
+        // Ex. https://tailwindcss.com/docs/optimizing-for-production#writing-purgeable-html
+        defaultExtractor: content => content.match(/[\w-/.:]+(?<!:)/g) || [],
```

Comment:
A comment as to what the regex is trying to match (with some examples) would be great.

Reply:
Ack, I merged before seeing this comment. I'll add a few words to the existing comment calling out colons and dots.